apps/core/scenes/search/PlacePicker/PlacePickerRefetchContainer.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+// This rule is disabled because this is just temporary the PlacePicker component.
+/* eslint-disable relay/unused-fields */
+
+import {
+  graphql,
+  createRefetchContainer,
+  type RelayRefetchProp,
+} from '@kiwicom/margarita-relay';
+import { debounce } from '@kiwicom/margarita-utils';
+import * as React from 'react';
+
+import PlacePickerContent from './PlacePickerContent';
+import type { PlacePickerRefetchContainer_locations as PlacePickerRefetchContainerType } from './__generated__/PlacePickerRefetchContainer_locations.graphql.js';
+
+type Props = {
+  locations: ?PlacePickerRefetchContainerType,
+  relay: RelayRefetchProp,
+};
+
+class PlacePickerRefetchContainer extends React.Component<Props> {
+  handleChangeText = debounce((text: string) => {
+    this.props.relay.refetch({ input: { term: text } });
+  }, 250);
```

Comment:
Just an idea – we could have numbers like this one in some config. Maybe some day we will need to create another debounced user input.

Reply:
Good idea.

Reply:
done